experimenter/experimenter/nimbus_ui/forms.py

```diff
@@ -1259,18 +1269,17 @@ def get_changelog_message(self):
         return f"{self.request.user} removed subscriber"
 
 
-class FeatureSubscribeForm(forms.ModelForm):
-    class Meta:
-        model = NimbusFeatureConfig
-        fields = []
+class FeatureSubscribeForm(FeatureSubscriberFormMixin):
+    def save(self, commit=True):
+        feature_config = super().save(commit=commit)
```

Comment:
Do we need to super().save here? We're not actually modifying the feature_config itself outside of the subscribers.